.circleci/config.yml

```diff
@@ -0,0 +1,103 @@
+version: 2.1
```

Comment:
would we want to leverage jenkins here too over circleci?

Reply:
also want to establish that we are for certain going to migrate to this new repository and deprecate the logdna node client rather than rename the logdna node client and add this pr there.

Reply:
Great point on Jenkins.   I would have gone that route, but circleci has a super-easy Orb that got us the code coverage badge, which I think is important now that we have 100%.

Yes, this is a new repo, and we will `npm deprecate` the other one.  The plan is to merge it Monday assuming it gets approved.